js/main.js

```diff
@@ -1,33 +1,106 @@
 /* global data */
 /* exported data */
-var entryForm = document.querySelector('.form');
-var photo = document.querySelector('img');
-var photoUrl = document.querySelector('.photoUrl');
-var title = document.querySelector('.title');
-var notes = document.querySelector('.notes');
-
-photoUrl.addEventListener('input', updatePhoto);
+var $entryForm = document.querySelector('.form');
+var $photo = document.querySelector('img');
+var $photoUrl = document.querySelector('.photoUrl');
+var $title = document.querySelector('.title');
+var $notes = document.querySelector('.notes');
+var $ul = document.querySelector('ul');
+var $viewElements = document.querySelectorAll('.view');
+var $entriesEmpty = document.querySelector('.entries-empty');
+var i;
 
+$photoUrl.addEventListener('input', updatePhoto);
 function updatePhoto(event) {
-  photo.setAttribute('src', photoUrl.value);
+  $photo.setAttribute('src', $photoUrl.value);
 }
 
-entryForm.addEventListener('submit', saveEntryValues);
-
+$entryForm.addEventListener('submit', saveEntryValues);
 function saveEntryValues(event) {
   event.preventDefault();
 
   var newEntry = {
     entryId: data.nextEntryId,
-    photoUrl: photoUrl.value,
-    title: title.value,
-    notes: notes.value
+    photoUrl: $photoUrl.value,
+    title: $title.value,
+    notes: $notes.value
   };
 
   data.nextEntryId++;
   data.entries.unshift(newEntry);
-  photo.setAttribute('src', 'images/placeholder-image-square.jpg');
-  photoUrl.value = '';
-  title.value = '';
-  notes.value = '';
+  $photo.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $photoUrl.value = '';
+  $title.value = '';
+  $notes.value = '';
+
+  for (i = 0; i < $viewElements.length; i++) {
+    if ($viewElements[i].getAttribute('data-view') !== 'entries') {
+      $viewElements[i].classList.add('hidden');
+    } else $viewElements[i].classList.remove('hidden');
+  }
+  data.view = 'entries';
```

Comment:
You have this same sort of functionality in multiple places, this would be a good opportunity to make a function definition whose sole job is to switch views, which receives an argument of a string with the name of the view to go to.